CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+You can find and compare releases at the GitHub release page.
+
+## [Unreleased]
+
+
+## [1.2.0] - 2021-11-16
+
+### Added
+- Dispatch Auth Events by @okaufmann in #45
```

Comment:
This one is interesting because technically it's category "Breaking Change" and should probably have been a major version bump, but that's too late anyway.